backend/ideas/models/ideas.py

```diff
@@ -0,0 +1,105 @@
+from django.db import models
+from django.contrib.auth.models import User
+from organization.models.organization import Organization
+from hubs.models.hub import Hub
+from location.models import Location
+
+
+def idea_image_path(instance, filename):
+    return "ideas/{}/image/{}".format(instance.id, filename)
+
+
+class Idea(models.Model):
+    name = models.CharField(
+        help_text="Name of an idea",
+        verbose_name="Name",
+        max_length=256
+    )
+
+    summary = models.CharField(
+        help_text="Summary of an idea",
+        verbose_name="Summary",
+        max_length=1024
+    )
+
+    description = models.TextField(
```

Comment:
I think we'll just have a short description? but not sure if that is 100% decided yet

Reply:
Removed